src/main/java/wromaciej/hvac_sim/display/Displayable.java

```diff
@@ -0,0 +1,6 @@
+package wromaciej.hvac_sim.display;
+
+public interface Displayable {
+
+    void display();
```

Comment:
btw where and what will implementation of that interface display?

Reply:
it is a sketch for views

Reply:
but shouldn't that return simply String with representation which should be later on shown? And then e.g. ConsoleView could use that String repr to print it to console, and HTMLView could use that String to print it to browser. 

The thing is Displayable should not really be responsible for printing anything.